docs/remote/troubleshooting.md

```diff
@@ -1076,13 +1076,11 @@ Extensions that access locally attached devices will be unable to connect to the
 
 If you run into an issue with one of the remote development extensions, it's important to collect the correct logs so that we'll be able to help [diagnose your issue](https://aka.ms/vscode-remote/issues/new).
 
-We recommend turning on trace level telemetry to get the most detailed logs: `F1` -> `Developer: Set Log Level...` -> `Trace`. From here, the type of logs depends on the context of your issue.
+Each remote extension has a command to view its logs. You can get the Remote - SSH logs with `F1` -> `Remote-SSH: Show Log`, which is the same as the output from the SSH Output channel: `F1` -> `Output: Focus on Output View`, and select "Remote - SSH" from the dropdown. Also please verify if you're able to SSH into your machine from an external terminal (not using Remote - SSH).
```

Comment:
I don't think it's necessary to mention both ways of getting to the same log.

Reply:
Sounds good - I removed the second way & just kept the first one (so that it's consistent with the other extensions).